kotsadm/pkg/store/ocistore/version_store.go

```diff
@@ -37,6 +37,44 @@ func (s OCIStore) appVersionConfigMapNameForApp(appID string) (string, error) {
 	return fmt.Sprintf("%s%s", AppVersionConfigmapPrefix, a.Slug), nil
 }
 
+func (s OCIStore) getLatestAppVersion(appID string) (*versiontypes.AppVersion, error) {
+	configMapName, err := s.appVersionConfigMapNameForApp(appID)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to get appversion config map name")
+	}
+
+	configMap, err := s.getConfigmap(configMapName)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to get app version config map")
```

Comment:
Should this be `ErrNotFound`?

Reply:
no, the config map will be created in the get request, so this is an error condition